tests/console/journalctl.pm

```diff
@@ -123,7 +123,7 @@ sub run {
     # Other configuration changes should be overridden using _drop-in_ file
     # To enable persistent logging in opensuse, we use systemd-logger.rpm that creates */var/log/journal/* directory
     get_current_boot_id \@boots;
-    if (is_opensuse) {
+    if (is_opensuse && !is_leap('>=15.3')) {
```

Comment:
The same change would be required on L#126, too.

Reply:
err.. it's line 126 :)

Reply:
haha, L#189 morning reviews before coffee :D